kani-compiler/src/codegen_cprover_gotoc/codegen/place.rs

```diff
@@ -242,54 +240,61 @@ impl<'tcx> GotocCtx<'tcx> {
     fn codegen_field(
         &mut self,
         parent_expr: Expr,
-        parent_ty_or_var: TypeOrVariant<'tcx>,
-        field: &FieldIdx,
-        field_ty_or_var: TypeOrVariant<'tcx>,
+        parent_ty_or_var: TypeOrVariant,
+        field_idx: FieldIdx,
```

Comment:
Is passing `field_idx` by value here intended?

Reply:
Yes. It's just an usize